PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,20 @@
 ## Change Summary
 
-> Description here
+> Link to Jira ticket if applicable
+> Describe the changes (even if there is a description in Jira)
+> Identify backwards incompatible changes; link to tickets/PRs that must merge or deploy first
+> Call out known issues, high risk areas, or future work to be done
+> Include guidance for reviewers if relevant
 
-## Type of change
-- [ ] Bug fix (fixes an issue)
-- [ ] New feature (adds functionality)
-- [ ] Refactor (changes code structure with same or similar functionality)
-- [ ] Libs update (Libs update with code change or with minor code change to get new version working)
+## How I know it works
```

Comment:
This could also be a useful place to link resources such as design specs, notion docs, etc. That doesn't need to be mandatory though.